spec/models/track_spec.rb

```diff
@@ -246,7 +354,8 @@
       self_organized_track.cfp_active = true
       self_organized_track.save!
       @a_track_organizer.add_role 'track_organizer', self_organized_track
-      @an_event_of_the_track = create(:event, program: self_organized_track.program, track: self_organized_track)
+      @an_event_of_the_track = create(:event, program: self_organized_track.program, track: self_organized_track, state: 'confirmed')
+      @schedule_of_the_track = create(:schedule, program: self_organized_track.program, track: self_organized_track)
```

Comment:
I would rename this as well.
schedule_of_self_organized_track
or
self_organized_track_schedule